probe/internal/cli/cli.go

```diff
@@ -0,0 +1,86 @@
+package cli
+
+import (
+	"context"
+	"os"
+	"os/signal"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/acs-fleet-manager/probe/pkg/runtime"
+)
+
+var (
+	// errInterruptSignal corresponds to a received SIGINT signal.
+	errInterruptSignal = errors.New("received interrupt signal")
+)
+
+// CLI defines the command line interface of the probe.
+type CLI struct {
+	runtime *runtime.Runtime
+}
+
+// New creates a CLI.
+func New() (*CLI, error) {
+	runtime, err := runtime.New()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create runtime")
+	}
+	return &CLI{runtime}, nil
```

Comment:
nit (to keep consistency with runtime.New()):
```suggestion
	return &CLI{runtime: runtime}, nil
```

Reply:
done